endpoints/apiconfig.go

```diff
@@ -769,19 +769,78 @@ func validateField(v reflect.Value, t reflect.StructField) error {
 	if err != nil {
 		return fmt.Errorf("parse tag: %v", err)
 	}
-	if v.Interface() != reflect.Zero(v.Type()).Interface() {
-		return nil
+
+	isZero := v.Interface() == reflect.Zero(v.Type()).Interface()
+	if isZero && tag.required {
```

Comment:
if `v` is an int type and is actually 0, wouldn't this trigger a false positive and return an error?
in other words, with this change, would i be able to pass a parameter p of value 0 and have it required at the same time?